docs/nui-components/Guide.md

```diff
@@ -0,0 +1,38 @@
+# Welcome to CommonUI
+This guide will take you through what you need to know to produce efficient, dynamic and visually rich applications in CommonUI easily
+
+## What is CommonUI
+Cover the common UI components of FHUB, TV, Mobile, Watch.
```

Comment:
No need to mention targets as want NUI to seem general and useful for multiple applications.